transit_odp/api/tests/test_avl_api.py

```diff
@@ -1,3 +1,4 @@
+from datetime import timedelta
```

Comment:
I would expect this to already be in dev from a main rebase

Reply:
Hi Paul,
One cannot commit directly into dev as the branch is protected hence this is going via a PR.